gnomad_qc/v4/assessment/calculate_per_sample_stats.py

```diff
@@ -414,6 +415,7 @@ def compute_agg_sample_stats(
     :param by_ancestry: Boolean indicating whether to stratify by ancestry.
     :param by_subset: Boolean indicating whether to stratify by subset. This is only
          working on "exomes" data.
+    :param data_type: Data type for sample QC metrics, e.g. "exomes" or "genomes".
```

Comment:
I don't think we need to add data_type, we can just check if by_subset is True, and then in main, if args.data_type and args.by_subset are both set, raise an error

Reply:
You're right, I did that.